ros2_control_demo_description/rrbot_description/gazebo/rrbot.gazebo.xacro

```diff
@@ -10,7 +10,7 @@ https://github.com/ros-simulation/gazebo_ros_demos/blob/kinetic-devel/rrbot_desc
     <!-- ros_control plugin -->
     <gazebo>
       <plugin name="gazebo_ros2_control" filename="libgazebo_ros2_control.so">
-        <parameters>$(find ros2_control_demo_bringup)/config/rrbot_gazebo_forward_controller_position.yaml</parameters>
```

Comment:
We do not need this file anymore and both gazebo and non-gazebo examples use the rrbot_controllers since it was intended to add the forward_position_controller to the gazebo example as well.